src/main/java/botbot/tasks/Deadline.java

```diff
@@ -0,0 +1,84 @@
+package botbot.tasks;
+
+import botbot.Parser;
+import botbot.exceptions.InvalidFormatException;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
+/**
+ * Represents a deadline with a description, deadline and completion status.
+ */
+public class Deadline extends Task {
+    public static final char TYPE_CODE = 'D';
+    public static final String FORMAT = "deadline <description> /by <D-M-YYYY HHmm> (eg. 17-3-2020 0945 "
```

Comment:
It could be beneficial to edit the name of this class variable to include more detail to explain what this format could be used for. 

For instance, maybe this constant could be renamed to "REQUIRED_FORMAT" to indicate that is the required format that users should use when providing input. 

I believe that the Event class has a class variable named "FORMAT" too that could benefit from this small edit too.

> Use name to explain - [Textbook](https://nus-cs2103-ay2021s1.github.io/website/book/codeQuality/nameWell/intermediate/useNameExplain/)